lib/samples/show_grid/show_grid_sample.dart

```diff
@@ -0,0 +1,368 @@
+//
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class ShowGridSample extends StatefulWidget {
+  const ShowGridSample({super.key});
+
+  @override
+  State<ShowGridSample> createState() => _ShowGridSampleState();
+}
+
+class _ShowGridSampleState extends State<ShowGridSample> {
+  // create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+  final _center = ArcGISPoint(
+      x: -7702852.905619,
+      y: 6217972.345771,
+      spatialReference: SpatialReference.webMercator);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Stack(
+          children: [
+            Column(
+              children: [
+                Expanded(
+                  // Add a map view to the widget tree and set a controller.
+                  child: ArcGISMapView(
+                    controllerProvider: () => _mapViewController,
+                    onMapViewReady: onMapViewReady,
+                  ),
+                ),
+                Row(
```

Comment:
When there is only one control, we've been using a Center instead of a Row, which simplifies a few things -- no need for mainAxisAlignment, and no need for an array.